pep_sphinx_extensions/pep_theme/static/style.css

```
@@ -32,7 +32,7 @@
 
 /* Set master colours */
 :root {
-    --colour-background: var(--light, white) var(--dark, #011);
+    --colour-background: var(--light, white) var(--dark, #121212);
```

```suggestion
    --colour-background: var(--light, white) var(--dark, #111);
```

`#111111` is basically identical, but keeps to the previous 3-character color granularity that all the other colors here have, and is a slightly closer match to the original (which the others were calibrated against) and with just a tiny bit more contrast, so seems reasonable to go with that.

---

I'm not too bothered about maintaining three characters, but sure, looks fine!